docs/sources/datasources/jaeger/_index.md

```diff
@@ -153,41 +152,55 @@ You can choose one of three options:
 You can define and configure the data source in YAML files as part of Grafana's provisioning system.
 For more information about provisioning, and for available configuration options, refer to [Provisioning Grafana]({{< relref "../../administration/provisioning#data-sources" >}}).
 
-#### Provisioning examples
-
-**Using basic auth and the trace-to-logs field:**
+#### Provisioning example
 
 ```yaml
 apiVersion: 1
 
 datasources:
   - name: Jaeger
     type: jaeger
-    uid: jaeger-spectra
+    uid: EbPG8fYoz
+    url: http://localhost:16686
     access: proxy
-    url: http://localhost:16686/
     basicAuth: true
     basicAuthUser: my_user
-    editable: true
+    readOnly: false
     isDefault: false
     jsonData:
-      tracesToLogs:
-        # Field with internal link pointing to a logs data source in Grafana.
-        # datasourceUid value must match the datasourceUid value of the logs data source.
+      tracesToLogsV2:
+        # Field with an internal link pointing to a logs data source in Grafana.
+        # datasourceUid value must match the uid value of the logs data source.
         datasourceUid: 'loki'
+        spanStartTimeShift: '1h'
+        spanEndTimeShift: '-1h'
         tags: ['job', 'instance', 'pod', 'namespace']
-        mappedTags: [{ key: 'service.name', value: 'service' }]
-        mapTagNamesEnabled: false
+        filterByTraceID: false
+        filterBySpanID: false
+        customQuery: true
+        query: 'method="${__span.tags.method}"'
+      # Note: tracesToLogs used before Grafana version 9.4
```

Comment:
Should we keep this? I mean if somebody uses <9.4 Grafana they should look at that version of docs so it's not lost, but I guess people should not use this anymore.

Reply:
No we shouldn't. Older documentation versions will suffice. Thank you :)